requirements.txt

```diff
@@ -1,11 +1,11 @@
-numpy==1.11.1
-pandas==0.18.1
-scipy==0.17.1
-scikit-learn==0.17.1
-scikit-image==0.12.3
-pillow==3.4.2
-matplotlib==1.5.1
-seaborn==0.7.0
+numpy
```

Comment:
Please add more information here, and fix this typo. Post from <a href='https://app.reviewnb.com?xyz=abc'>ReviewNB</a>